### PR TITLE
Implement iterator method in ServerListPingEvent

### PR DESCRIPTION
**Pull request by kamcio96**
Test code:

```
@Override
public void onEnable() {
    Bukkit.getPluginManager().registerEvents(this, this);
}

@EventHandler
public void onPing(ServerListPingEvent event) {
    System.out.println("Online:");
    Iterator<Player> i = event.iterator();
    int a=0;
    while (i.hasNext()) {
        System.out.println(++a + " " + i.next());
    }

    i = event.iterator();
    try {
        while (i.hasNext()) {
            i.remove();
        }
    } catch (Exception e) {
        System.out.println("Exception: " + e.getMessage());
    }
}
```

Log:

```
13:26:03 [INFO] Online:
13:26:03 [INFO] 1 GlowPlayer{name=kamcio96}
13:26:03 [INFO] Exception: null
```

### CODE REVIEW

**Comment by mastercoms** [comment]
is there a limit to how many players that can be in the iterator?

**Comment by kamcio96** [comment]
no, should be?

**Comment by deathcap** [comment]
I think it's supposed to be unlimited, since it returns all of the online players in order to allow plugins to hide specific online players: https://github.com/GlowstonePlusPlus/Glowkit/blob/bukkit%2Bglowkit/src/main/java/org/bukkit/event/server/ServerListPingEvent.java

```java
    /**
     * {@inheritDoc}
     * <p>
     * Calling the {@link Iterator#remove()} method will force that particular
     * player to not be displayed on the player list, decrease the size
     * returned by {@link #getNumPlayers()}, and will not be returned again by
     * any new iterator.
     *
     * @throws UnsupportedOperationException if the caller of this event does
     *     not support removing players
     */
    @Override
    public Iterator<Player> iterator() throws UnsupportedOperationException {
        throw new UnsupportedOperationException();
    }
```

@kamcio96 can you test, does this PR let plugins remove players from the iterator?

**Comment by mastercoms** [comment]
Spigot has a config option to limit or randomize players, I think.

**Comment by kamcio96** [comment]
So should it support remove() or not?

**Comment by mastercoms** [comment]
It should support remove().

File: src/main/java/net/glowstone/net/handler/status/StatusRequestHandler.java [hunk 1]
```diff
@@ -1,18 +1,21 @@
 package net.glowstone.net.handler.status;
 
 import com.flowpowered.networking.MessageHandler;
+import com.google.common.collect.Iterators;
```
Comment (mastercoms) [inline src/main/java/net/glowstone/net/handler/status/StatusRequestHandler.java]: Unused import :)

**Comment by kamcio96** [comment]
Done

**Pull request merged**